app/routes.py

```diff
@@ -1,2 +1,242 @@
-from flask import Blueprint
+from flask import request, jsonify, abort, Blueprint, g
+from app import db
+from sqlalchemy import exc
+from app.models.task import Task
+from app.models.goal import Goal
 
+task_bp = Blueprint("tasks", __name__, url_prefix="/tasks")
+goal_bp = Blueprint("goals", __name__, url_prefix="/goals")
+
+
+@goal_bp.before_request
+@task_bp.before_request
+def get_model_and_label():
```

Comment:
Awesome to see how the names of the `Blueprint` can be useful. I hadn't see the `before_request` implemented before. Nice work.